core-web/libs/template-builder/src/lib/components/template-builder/components/template-builder-box/template-builder-box.component.html

```diff
@@ -46,7 +46,9 @@
     </div>
     <p-scrollPanel [style]="{ width: '100%', height: '10.9375rem' }">
         <div class="template-builder-box__item" *ngFor="let item of items; let i = index">
-            <p>{{ item.identifier }}</p>
+            <p data-testId="container-title">
+                {{ containerMap[item.identifier]?.title || item.identifier }}
```

Comment:
What happen if we don't pass `containerMap`? we sure it doesn't break?

Reply:
`containerMap` will always be passed in current scenarios as it is always initialized as empty by the template builder store. Could put some checks if you think we may change this approach in the future.

Reply:
We can assume that a library will always be pass all the required inputs, we had problems in the past. Angular 16 introduce required inputs but for now, is our job to manage those scenarios manually.